packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -1000,3 +1000,30 @@ Posts.addView("reviews2018", terms => {
 })
 // We're filtering on nominationCount greater than 2, so do not need additional indexes
 // using nominations2018
+
+Posts.addView("tagProgressTagged", terms => {
+  return {
+    selector: {  
+      tagRelevance: {$exists: true},
+      baseScore: {$gt: 25},
+      $expr: {
+          $gt: [
+              {$size: 
+                  {$filter: {
+                      input: {$objectToArray: "$tagRelevance"},
+                      cond: {$not: {$in: ["$$this", ["xexCWMyds6QLWognu", "sYm3HiWcfZvrGu3ui", "izp6eeJJEg9v5zcur", "fkABsGCJZ6y9qConW", "Ng8Gice9KNkncxqcj", "MfpEPj6kJneT9gWT6", "3uE2pXvbcnS9nnZRE"]]}}
```

Comment:
Put in a comment explaining that this is a temporary hack, and what these IDs are. (Among other reasons, this code would not work on EA Forum).